1.0.0-rc.0 not installing via NPM on Windows 10

### Expected behavior
lisk-elements@1.0.0-rc.0 installs via NPM successfully on Windows 10.

### Actual behavior
Installation of lisk-elements@1.0.0-rc.0 fails during installation via NPM on Windows 10. The following error code is shown:

```
> lisk-elements@1.0.0-rc.0 postinstall C:\projectdirectory\node_modules\lisk-elements
> if [ ! -f ./dist-node/index.js ]; then npm run build; fi

! was unexpected at this time.
...
npm ERR! Failed at the lisk-elements@1.0.0-rc.0 postinstall script.
```

### Steps to reproduce
`npm i --save lisk-elements@1.0.0-rc.0`

(I did try `npm i --save lisk-elements` but it installed version 0.0.0)

### Which version(s) does this affect? (Environment, OS, etc...)
OS: Windows 10 Home
Node: v8.11.1
NPM: v5.8.0

--- a/.gitignore
+++ b/.gitignore
@@ -52,9 +52,8 @@ jspm_packages
 tmp
 
 # Distribution folders
-dist-browser/*.js
-dist-browser/*.json
-dist-node
+dist-browser/
+dist-node/
 
 # Built test files
 browsertest/package.json

--- a/README.md
+++ b/README.md
@@ -37,7 +37,7 @@ import { APIClient, transactions } from 'lisk-elements';
 const { APIClient, transactions } = require('lisk-elements');
 ```
 
-**Note:** If you are installing Lisk Elements as an npm dependency via a GitHub reference, you will need to manually build the distribution files by running the following commands:
+**Note:** If you are installing Lisk Elements as an npm dependency via a GitHub reference, you will need to manually build the distribution files by running the following commands from the root directory of your project:
 
 ```
 cd node_modules/lisk-elements
@@ -59,6 +59,8 @@ Or minified:
 ```
 
 ### Installation from source
+
+Our source code is hosted on GitHub. You can build the distribution yourself by cloning the repository, installing the relevant dependencies and running our build script as follows:
 
 ```
 git clone https://github.com/LiskHQ/lisk-elements.git

--- a/README.md
+++ b/README.md
@@ -9,7 +9,7 @@ Lisk Elements is a JavaScript library for [Lisk][Lisk Core GitHub], the blockcha
 
 ## Installation
 
-## Installation via npm
+### Installation via npm
 
 Add Lisk Elements as a dependency of your project:
 
@@ -37,6 +37,13 @@ import { APIClient, transactions } from 'lisk-elements';
 const { APIClient, transactions } = require('lisk-elements');
 ```
 
+**Note:** If you are installing Lisk Elements as an npm dependency via a GitHub reference, you will need to manually build the distribution files by running the following commands:
+
+```
+cd node_modules/lisk-elements
+npm run build
+```
+
 ### Installation via CDN
 
 Include the following script using the following HTML. The `lisk` variable will be exposed.
@@ -49,6 +56,15 @@ Or minified:
 
 ```html
 <script src="https://js.lisk.io/lisk-elements-1.0.0.min.js"></script>
+```
+
+### Installation from source
+
+```
+git clone https://github.com/LiskHQ/lisk-elements.git
+cd lisk-elements
+npm install
+npm run build
 ```
 
 ## Usage

--- a/package.json
+++ b/package.json
@@ -59,7 +59,6 @@
     "build:check": "node -e \"require('./dist-node')\"",
     "precommit": "lint-staged && npm run lint",
     "prepush": "npm run lint && npm test",
-    "postinstall": "if [ ! -f ./dist-node/index.js ]; then npm run build; fi",
     "prepublishOnly": "rm -r ./node_modules && npm install && npm run prepush && npm run build"
   },
   "dependencies": {

